app/controllers/concerns/blacklight/catalog.rb

```diff
@@ -16,10 +16,6 @@ module Blacklight::Catalog
       helper_method :sms_mappings, :has_search_parameters?
     end
 
-    # The index action will more than likely throw this one.
-    # Example: when the standard query parser is used, and a user submits a "bad" query.
-    rescue_from Blacklight::Exceptions::InvalidRequest, with: :handle_request_error
```

Comment:
How about adding this to the catalog_controller.rb template file, but keep it commented out?  This can show people how they can rescue this error if they want to.

Reply:
Sure why not. That's `lib/generators/blacklight/templates/catalog_controller.rb`?

Reply:
Would you put the `handle_request_error` method example definition in there too @jcoyne , or just the `rescue_from`?  The current definition is maybe a bit hairy.

Reply:
Yeah, I'd just put the rescue from in there.

Reply:
done, good idea, thanks!